### ISSUE TITLE
Question about cti-taxii-server usage

### ISSUE THREAD

**Issue by tsitsiflora** [oasis-open/cti-taxii-server#115]
Hi everyone,

I am pretty new to cyber threat intelligence and would like your help. I managed to set up the taxii server and it's running on 127.0.0.1:5000. What I am not sure of is what the server is supposed to display once it starts running. I am getting a 404 error. The documentation doesn't really accommodate beginners and I would really appreciate it if someone provides insight on what to do after you have the server up and running.

I tried some of the endpoints mentioned here https://github.com/oasis-open/cti-taxhttps://github.com/oasis-open/cti-taxii-clientii-client. I just don't know if I'm doing the right thing or not.

Thank you in advance!

**Comment by zrush-mitre** [comment]
Hi @tsitsiflora and thank you for checking out the taxii server!

We need a lot more information in order to diagnose your problem. Are you following the steps from the readme to create the server? How are you trying to request objects from the server? Are you seeing the requests show in the server's logs?

Thanks.

**Comment by tsitsiflora** [comment]
Hi @zrush-mitre, thank you for replying.

Yes I followed the tutorial from the readme. Please see below my config file:

```
{
    "backend": {
        "module": "medallion.backends.memory_backend",
        "module_class": "MemoryBackend",
        "filename": "./test.json"},

    "users": {
       "admin": "Password0",
       "user1": "Password1",
       "user2": "Password2"},
    "taxii": {
       "max_page_size": 100
}


}
```
After running : medallion medallion_config.json this is what I get on my terminal

```
onfig.json 
 * Serving Flask app "medallion" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
127.0.0.1 - - [07/May/2020 18:02:50] "GET / HTTP/1.1" 404 -
127.0.0.1 - - [07/May/2020 18:02:51] "GET /favicon.ico HTTP/1.1" 308 -
127.0.0.1 - - [07/May/2020 18:02:51] "GET /favicon.ico/ HTTP/1.1" 401 -
127.0.0.1 - - [07/May/2020 18:03:31] "GET /favicon.ico/ HTTP/1.1" 404 -
127.0.0.1 - - [07/May/2020 18:03:46] "GET / HTTP/1.1" 404 -
```
I am trying to request the objects through the web url, by just editing the url. But all the urls I am trying give me 404. I am following instructions from this client as well https://github.com/oasis-open/cti-taxii-client .

In summary:
1. I want to get objects but I keep getting 404
2. I also want to be able to post objects to the server

Thank you for your help.

**Comment by zrush-mitre** [comment]
Hi @tsitsiflora,

Can you post the code you are using to access the server? 

You aren't trying to access the server through a web browser right? The Python server implementation is not made to be access by a browser, but rather as an API.

**Comment by tsitsiflora** [comment]
Hi @zrush-mitre ,

Sure. This is the code that I'm using:
```
from taxii2client.v20 import Server
server = Server('http://127.0.0.1:5000/taxii/', user='admin', password='Password0')
api_root = server.api_roots[0]
collection = api_root.collections[0]
objects = collection.get_objects()

print(api_root)
print(collection.title)
print(objects)
```
and from the console. Some of the endpoints I've been trying all return 404.:
```
* Serving Flask app "medallion" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
127.0.0.1 - - [09/May/2020 17:59:35] "GET / HTTP/1.1" 404 -
127.0.0.1 - - [09/May/2020 18:01:05] "GET /api/v1/ HTTP/1.1" 404 -
127.0.0.1 - - [09/May/2020 18:01:26] "GET /api/ HTTP/1.1" 404 -
127.0.0.1 - - [09/May/2020 18:02:09] "GET /taxii/ HTTP/1.1" 404 -

```

**Comment by zrush-mitre** [comment]
Hi @tsitsiflora,

Check that your config that is running with medallion is really the one you want. I noticed you change the data_file value in the config, so make sure that is referencing the file you think it does.

Also, look in the data_file you are using and call one of the api_roots directly with the Server object. Then try printing one of it's attributes. For example: if api1 is in your data_file, use:

`server = Server('http://127.0.0.1:5000/api1', user='admin', password='Password0')
print(server.title)`

**Comment by tsitsiflora** [comment]
Thank you for your help!
